autopts/ptsprojects/stack.py

```diff
@@ -1109,6 +1147,10 @@ def is_svc_supported(self, svc):
             "MESH":         0b0010000,
             "MESH_MMDL":    0b0100000,
             "GATT_CL":      0b1000000,
+            "VCS":          0b1000001,
```

Comment:
those are bitfields (bit number match service ID)

        Opcode 0x02 - Read Supported Services command/response

                Controller Index:       <non-controller>
                Command parameters:     <none>
                Response parameters:    <supported services> (variable)

                Each bit in response is a flag indicating if service with ID
                matching bit number is supported. Bit set to 1 means that
                service is supported. If specific bit is not present in response
                (less than required bytes received) it shall be assumed that
                service is not supported.

Reply:
done